src/components/widgets/TabsDynamic.vue

```diff
@@ -0,0 +1,112 @@
+<script setup lang="ts">
+import { ref, computed, onMounted, onUnmounted } from 'vue';
+
+import { isCveValid } from '@/utils/helpers';
+
+const props = defineProps<{
+  addableItems?: any[];
+  labels: string[];
+  tabProps: Record<string, any>;
+}>();
+
+const emit = defineEmits<{
+  'add-tab': [value?: any];
```

Comment:
could we use 'tab:add' just for format consistency with other emit names?

Reply:
Since tab isn't a model entity, the convention may not apply, but I don't feel strongly